src/main/kotlin/no/nav/syfo/behandlerdialog/api/BehandlerdialogApi.kt

```diff
@@ -0,0 +1,43 @@
+package no.nav.syfo.behandlerdialog.api
+
+import io.ktor.http.*
+import io.ktor.server.application.*
+import io.ktor.server.request.*
+import io.ktor.server.response.*
+import io.ktor.server.routing.*
+import no.nav.syfo.application.api.VeilederTilgangskontrollPlugin
+import no.nav.syfo.behandlerdialog.api.domain.BehandlerdialogDTO
+import no.nav.syfo.client.veiledertilgang.VeilederTilgangskontrollClient
+import no.nav.syfo.domain.PersonIdent
+import no.nav.syfo.util.*
+
+const val behandlerdialogApiBasePath = "/api/internad/v1/behandlerdialog"
+const val behandlerdialogApiPersonidentPath = "/personident" // TODO: Noe annet enn /personident eller?
```

Comment:
Akkurat dette har jeg og JML irritert oss litt over, så kanskje vi skal komme på noe annet? Gjerne kanskje sånn at det passer for både GET og POST-kallet? Selv om POST er én ny melding, og GET er en liste med meldinger da 🤔

Reply:
Enig i at vi godt kan bruke noe annet. Jeg synes `/melding` passer fint på POST (man oppretter en ny melding i en behandlerdialog). 
Kan man bruke bare base-urlen for GET (det er vel snakk om å hente all "behandlerdialog" for enn person?). Kommer ikke på noe som passer godt for både POST og GET, men er det ikke greit om de er forskjellig?

Reply:
Nå ble det `/melding` på POST og basepath på get 👍🏼